### DISCUSSION TITLE
Error with a scene

### DISCUSSION THREAD

**Discussion by gdambic** [jeertmans/DiffeRT2d#91] [Q&A] [answered]
Hello,

I get an error when trying to use a simple scene and I am not sure what I am doing wrong. Here are the steps to reproduce:
1. Create a new venv, Python 3.12
2. Install differt2d: pip install differt2d
3. Add code:
```
from differt2d.scene import Scene
scene = Scene.square_scene()
```
When run, I get:
```
C:\path\.venv\Scripts\python.exe C:\path\main.py 
Traceback (most recent call last):
  File "C:\path\main.py", line 3, in <module>
    scene = Scene.square_scene()
            ^^^^^^^^^^^^^^^^^^^^
  File "C:\path\.venv\Lib\site-packages\differt2d\scene.py", line 742, in square_scene
    tx = Point(xy=jnp.asarray(tx_coords, dtype=float))
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\path\.venv\Lib\site-packages\equinox\_module.py", line 594, in __call__
    object.__setattr__(self, "__class__", cls)
TypeError: __class__ assignment: 'Point' object layout differs from 'Point'
```

What am I doing wrong?

**Comment by jeertmans** [comment, marked as answer]
Hi @gdambic, thanks for reporting your issue!

As this is a bug, I moved the discussion to #92. Can you please provide information about your Python environment there?

I am locking this conversation so we avoid commenting in too many places :-)
